src/MeshKernelNET/Native/Mesh1DNative.cs

```diff
@@ -27,9 +27,19 @@ public struct Mesh1DNative
         /// </summary>
         public int num_nodes;
 
+        /// <summary>
```

Comment:
the number of valid nodes can be computed, there is no need to make it a member.

Reply:
we discussed this briefly. This is part of the native meshkernel API, meshkernel is already computing these values for its clients